go-runtime-builder/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
```

Comment:
Nit.  For consistency, let's add # even for empty lines on the license header throughout.